index.js

```diff
@@ -57,6 +57,63 @@ bot.registerCommand(
     }
 );
 
+bot.registerCommand(
+    "createPoll",
+    (msg, args) => {
+
+        const emojiSet = args[0];
+        let pollTitle = [];
+        let optionSet = [];
+        let optionBuilder = [];
+        let argCount = 0;
+
+        args.shift(); // Removes the first option that determins what emojis to use
+        args.push('99'); // Completely arbitrary number chosen to indicate the end of all the options. This is a bad idea and I need to figure out a different way to break up the options later.
+
+        for (let arg of args){
+            argCount++
+            if (arg === "Description:" ){
+                break;
+            }
+            pollTitle.push(arg);
+        }
+
+        args.splice(0, (argCount)); // Removes the Title
+
+        args.forEach( arg => { // Handles formating the discription
```

Comment:
I know I said to use `forEach` the other day, but I realize now that this would probably be better written as

```
for (const arg of args) {
  ...
}
```

There are also some tricks that you can do to do this in a function style, like with `reduce`, but there is no reason to complicate this with them.